/*
 * This file was automatically generated by EvoSuite
 * Sun Aug 12 19:45:10 KST 2018
 */

package com.substanceofcode.utils;

import static org.junit.Assert.assertEquals;
import org.junit.AfterClass;
import org.junit.BeforeClass;
import static org.evosuite.runtime.EvoAssertions.*;

import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.Test;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(useVNET = true, separateClassLoader = true, useJEE = true) 
public class CauseRecStoreException_ESTest3 extends CauseRecStoreException_ESTest_scaffolding3 {
@BeforeClass
public static void initEvoSuiteFramework() {
String a = "aa";
}
@AfterClass
public static void exitEvoSuiteFramework() {
String a = "aa";
}

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      CauseRecStoreException causeRecStoreException0 = new CauseRecStoreException("", (Throwable) null);
      assertEquals("", causeRecStoreException0.getMessage());
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      CauseRecStoreException causeRecStoreException0 = new CauseRecStoreException("");
      assertEquals("", causeRecStoreException0.getMessage());
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      CauseRecStoreException causeRecStoreException0 = new CauseRecStoreException();
      assertEquals("com.substanceofcode.utils.CauseRecStoreException", causeRecStoreException0.toString());
  }
}
